autogen/main/cluster.tf.tmpl

```diff
@@ -52,6 +52,14 @@ resource "google_container_cluster" "primary" {
       channel = release_channel.value.channel
     }
   }
+{% if beta_cluster %}
+  dynamic "confidential_nodes" {
+    for_each = local.confidential_node_config
+    content {
+      enabled = confidential_nodes.value.enabled
```

Comment:
For future reference, it's usually more intuitive to just hard-code this (`enabled = true`).

Reply:
I thought this would be more aligned with existing code like
https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/blob/8e92f6ec0d71bfffee9fa3621b55bbbd9091b0d0/autogen/main/cluster.tf.tmpl#L43

Reply:
Yes, that's an anti-pattern though.